src/resources/projects/components/EditForm/index.tsx

```diff
@@ -0,0 +1,179 @@
+  attributes = [],
+  children,
+}) => {
+  const notify = useNotify();
+  const redirect = useRedirect();
+  const [projectProfessionals, setProjectProfessionals] =
+    useState(professionals);
+  const [projectAttributes, setProjectAttributes] = useState(
+    parseProjectAttributes(attributes),
+  );
+  const {
+    data: allProfessionals,
+    isLoading: isLoadingProjects,
+    error: errorProjects,
+  } = useGetList<ProfessionalResponse>('professionals');
+
+  if (isLoadingProjects) {
+    return <Loading />;
+  }
+
+  if (errorProjects) {
+    notify(`Error: ${errorProjects}`, { type: 'error' });
+    redirect('list', 'projects');
+    return null;
+  }
+
+  const handleSave = (values: unknown) => {
+    const parsedValues = values as ParsedProjects;
+    const editProfessionalRequest: ProjectRequest = {
+      id: parsedValues.id,
+      name: parsedValues.name,
+      from: parsedValues.from,
+      to: parsedValues.to,
+      duration: parsedValues.duration,
+      description: parsedValues.description,
+      attributes: Object.values(projectAttributes)
+        .flat()
+        .map((attribute: ParsedProjectAttribute) => ({
+          attributeId: attribute.id,
+          attribute: attribute,
+          from: new Date(parsedValues.from),
+          to: new Date(parsedValues.to),
+        })),
+      professionals: projectProfessionals.map((professional) => ({
+        professionalId: professional.professional.id,
+        professional: professional.professional,
+        responsibility: professional.responsibility,
+      })),
+    };
+
+    save(editProfessionalRequest, parsedValues.id);
+  };
+
+  return (
+    <SimpleForm
+      component={Box}
+      onSubmit={handleSave}
+      toolbar={
+        <Toolbar>
+          <SaveButton alwaysEnable />
+        </Toolbar>
+      }
+    >
+      <Box sx={{ display: 'flex', width: '100%' }}>
+        <Card variant="outlined" sx={{ m: 0, mr: 1, p: '25px', flex: 1 }}>
+          {children}
+          <Box
+            sx={{
+              display: 'flex',
+              justifyContent: 'space-between',
+            }}
+          >
+            <TextInput
+              source={addSource<ProjectRequest>('name')}
+              sx={{ flex: 1, mr: 1 }}
+              validate={required()}
+            />
+          </Box>
+          <Box
+            sx={{
+              display: 'flex',
+              justifyContent: 'space-between',
+              alignItems: 'center',
+            }}
+          >
+            <TextInput
+              validate={required()}
+              source={addSource<ProjectRequest>('description')}
+              sx={{ flex: 1, mr: 1 }}
+            />
+          </Box>
+          <Box
+            sx={{
+              display: 'flex',
+              justifyContent: 'space-between',
+            }}
+          >
+            <DateInput
+              validate={required()}
+              source={addSource<ProjectRequest>('from')}
+              sx={{ flex: 1, mr: 1 }}
```

Comment:
```suggestion
              sx={{ flex: 1, mr: 1 }}
              format={(value) => {
                if (!value) {
                  return '';
                }
                return formatDate(value);
              }}
```
We need to format the timestamp date from the response to 'YYYY-MM'DD'